package.json

```diff
@@ -98,6 +98,7 @@
     "openzeppelin-solidity": "^2.2",
     "set-protocol-utils": "^1.0.0-beta.32",
     "tiny-promisify": "^1.0.0",
+    "truffle-flattener": "^1.4.0",
```

Comment:
Not sure you meant to have this in here but I'm ok having it part of our packages since we use it with some regularity